test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -288,6 +290,16 @@ private static boolean isOperatorVersionAbove040() {
         }));
     }
 
+    // Removal of the JVM option 'UseCGroupMemoryLimitForHeap' is required with JVM environments >= 10. This option
+    // is supplied by default by the operators. We cannot 'deactive' it using the XX:- counterpart as it is unrecognized.
+    private String[] getSegmentStoreJVMOptions() {
+        return new String[]{"-XX:+UseContainerSupport", "-XX:+IgnoreUnrecognizedVMOptions"};
+    }
+
+    private String[] getControllerJVMOptions() {
+        return new String[]{"-XX:+UseContainerSupport", "-XX:+IgnoreUnrecognizedVMOptions"};
```

Comment:
Why do we need IgnoreUnrecognizedVMOptions?

Reply:
The Pravega Operator supplies certain default JVM options (`+UseCGroupMemoryLimitForHeap`), which are not recognized by later JVMs. Any kubernetes deployment wouldn't work until the Pravega Operator is adjusted accordingly.